simpleocr/files.py

```diff
@@ -10,21 +10,8 @@
 GROUND_EXTENSIONS_DEFAULT = GROUND_EXTENSIONS[0]
 
 
-def open_img(path_or_name):
```

Comment:
Personally, I'd prefer it if the code was *in* the class, because it's only use is when the class is used, and nowhere else in the program. It there any other reason for wanting to keep it from the class, except for keeping the class code minimal? (Maybe this is a stupid question, but is it wrong to have a big class?)